docs/components/framework_components/skpager.md

```diff
@@ -0,0 +1,52 @@
+# SKPager
+
+Show a [ViewPager2](https://developer.android.com/jetpack/androidx/releases/viewpager2) as described in Android documentation 
+
+## Usage
+In your ScreenVC, declare a SKPagerVC. You also have to declare screens you want to display in `SKUses` annotation :
+
+```kotlin
+@SKUses([PaymentSlideScreenVC::class])
+interface PaymentOnBoardingScreenVC : SKScreenVC {
+    val pager: SKPagerVC
+}
+```
+
+Launch the generation with the SKGenerate gradle task.
+In your screen, you have to instanciate your component as below :
+
+```kotlin
+override val pager = SKPager(
+        initialScreens = listOf(PaymentSlideScreen(/**/), PaymentSlideScreen(/**/)),
+        initialSelectedPageIndex = 0,
+        swipable = true
+    )
+```
+
+In your xml layout file, you have to declare a `ViewPager2` widget with id `page` : 
+
+```xml
+<androidx.viewpager2.widget.ViewPager2
+    android:id="@+id/pager"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    app:layout_constraintBottom_toTopOf="@+id/linear_button"
+    app:layout_constraintTop_toBottomOf="@+id/actionBar" />
+```
+
+`initialScreens` could be an empty list. You can set screens to display after SKPager init, setting `screens` properties : 
```

Comment:
je propose
The `initialScreens` property can be an empty list. You can define the list of TabPages to be displayed at any time by setting the screens property.